manifests/kube-state-metrics-clusterRoleBinding.yaml

```diff
@@ -3,7 +3,7 @@ kind: ClusterRoleBinding
 metadata:
   labels:
     app.kubernetes.io/name: kube-state-metrics
-    app.kubernetes.io/version: 1.9.5
+    app.kubernetes.io/version: v1.9.5
```

Comment:
This is wrong and needs a fix in KSM. Not blocking this PR though.

Reply:
I believe we fixed it master IIRC and not in 1.9 branch? 🤔

Reply:
as discussed OOB this discussion can be lead on KSM and is not blocking this PR.